lib/css/negativeMargin/negativeMarginBottom.css.ts

```diff
@@ -0,0 +1,36 @@
+import { style, styleVariants } from '@vanilla-extract/css';
```

Comment:
There's lots of repetition in these files. Would we be able to put them in Sprinkles instead? That or just combine them and make a function to generate each of the directions?